README.md

```diff
@@ -6,6 +6,11 @@ List of GitHub profiles that have awesome customisation, that you can use for in
 
 Feel free to submit a pull request with your suggestions of awesome github profiles.
 
+[![made with love](https://forthebadge.com/images/badges/built-with-love.svg)](https://github.com/EddieHubCommunity)
+[![made with markdown](https://forthebadge.com/images/badges/made-with-markdown.svg)](https://github.com/EddieHubCommunity)
+[![open source](https://forthebadge.com/images/badges/open-source.svg)](https://github.com/EddieHubCommunity)
```

Comment:
Maybe you could link this one to an open source guide or something similar to what the badge is related to (same with the other ones). Just an idea. 🙂